src/amo/components/Addon/styles.scss

```diff
@@ -202,6 +202,7 @@
       // This span makes sure the left widget does not move vertically when
       // there is a long description.
       grid-row: 1 / span 100000;
+      overflow-x: hidden;
```

Comment:
Is this needed for the same as line 222?

Reply:
No, this isn’t Safari-related. It happens on Firefox.